Builder/Admin/ListBuilder.php

```diff
@@ -223,4 +229,47 @@ protected function findBatchActions()
             $this->addBatchAction($action);
         }
     }
+
+  /**
+   * Return a list of actions from excel.export
+   * 
+   * @return array
+   */
+  public function getExcelActions()
+  {
+      if (null === $this->excelActions) {
+          $this->excelActions = array();
+          $this->fillExportActions();
+      }
+
+      return $this->excelActions;
+  }
+
+  protected function fillExportActions()
+  {
+      $export = $this->getGenerator()->getFromYaml('builders.excel.params.export', []);
+      if (!count($export)) return;
+
+      foreach ($export as $keyName => $params ) {
+          if (isset($params['show_button']) && filter_var($params['show_button'], FILTER_VALIDATE_BOOLEAN)) {
+              $action = new ExcelAction($keyName, $this);
+              $action->setClass($this->getExportParamsForKey($keyName, 'class', 'btn-info'));
+              $action->setIcon($this->getExportParamsForKey($keyName, 'icon', 'fa-file-excel-o'));
+              $action->setLabel($this->getExportParamsForKey($keyName, 'label', 'action.generic.excel'));
+              $this->excelActions[$keyName] = $action;
+          }
+      }
+  }
+
+  public function getExportParamsForKey($key, $name, $default)
+  {
+      if (!$key) return $default;
+
+      $export = $this->getGenerator()->getFromYaml('builders.excel.params.export', []);
+      if (!count($export)) return $default;
+
+      if (!isset($export[$key]) || !count($export[$key]) || !isset($export[$key][$name])) return $default;
+
+      return $export[$key][$name];
+  }
```

Comment:
Why is this code located in the `ListAction`? It is not used (or should not be used) there...